src/components/Header/Header.module.scss

```diff
@@ -0,0 +1,160 @@
+.header {
+  &__content {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    margin: 0 260px;
+  }
+
+  &__add {
+    background: var(--primary);
+    height: 42px;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    text-align: center;
+    padding: 0 15px;
+
+    &-text {
+      font-style: normal;
+      font-weight: var(--weight-bolder);
+      font-size: 11px;
+      line-height: 18px;
+
+      letter-spacing: 2.2px;
+      text-transform: uppercase;
+
+      color: var(--white);
+    }
+  }
+
+  &__languages {
+    justify-self: start;
+  }
+
+  &__logo {
+    justify-self: center;
+    height: 33px;
+    width: 264px;
+  }
+
+  &__main {
+    height: 120px;
+    min-width: 100%;
+    display: grid;
+    grid-template-columns: 100px 264px 100px;
+    justify-content: space-between;
+    align-items: center;
+    grid-template-columns: repeat(3, 120px);
+  }
+
+  &__links {
+    overflow: scroll;
+    max-width: 100%;
+    ul {
+      display: flex;
+      li:not(:first-child) {
+        margin-left: 40px;
+      }
+    }
+  }
+
+  &__buttons {
+    justify-self: end;
+    display: flex;
+    align-items: center;
+    button:not(:first-child) {
+      margin-left: 30px;
+    }
+  }
+
+  &__line {
+    width: 100%;
+    height: 2px;
+    background: var(--gray-9);
+  }
+}
+
+@media screen and (max-width: 1440px) {
```

Comment:
Please do it as scss mixin. Refactor to match media queries of the Figma mockup

Reply:
in some cases horizontal scroll appears, so I should set another media queries' width